app/code/Magento/Sales/Api/Data/TrackUrlInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Sales\Api\Data;
+
+/**
+ * Shipment Track Creation interface.
+ */
+interface TrackUrlInterface
```

Comment:
Introduction of the new interface should bump the minor version of a package.

Reply:
I've bumped the version number

Reply:
Do not see any changes in composer.json file.

Reply:
Am I supposed to update this composer.json?

https://github.com/RhodriOwainDavies/magento2/blob/0e13ba15b73fc701029ccbd7b3f30863c4e640fd/app/code/Magento/Sales/composer.json

Can you confirm what you mean exactly?

Reply:
You need to bump the version of composer.json in Sales module and specify the updated version in the `replace` section of "root" composer.json file. Also `TrackUrlInterface` should be marked as API.

Reply:
> need to bump the version of `composer.json` in Sales module and specify the updated version in the replace section of "root" `composer.json` file

Would be nice if such infor added to http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/ or http://devdocs.magento.com/guides/v2.2/extension-dev-guide/versioning/